scripts/chart-test.sh

```diff
@@ -34,7 +34,7 @@ fi
 # Runs chart-testing (ct CLI) with remote flag to avoid git errors
 
 ct lint --remote "$REMOTE" --config ct.yaml || exit 1
-rm charts/pelorus/charts/*.tgz
+rm charts/pelorus/charts/*.tgz &> /dev/null
```

Comment:
changing to use flag `-f, --force           ignore nonexistent files and arguments, never prompt`.

Added this because I have run it once and the tgz file was not created and command complained the file did not exist

Reply:
right, so you could use `-f`, but if the file can not be removed even with `-f` flag we won't know why. As I wrote, wrong owner or permissions that prevents to remove the file by the current user. Not big deal as it probably won't happen with the normal usage.